packages/mendel-pipeline/src/cache/index.js

```diff
@@ -8,9 +8,12 @@ const variationMatches = require('mendel-development/variation-matches');
 class MendelCache extends EventEmitter {
     constructor(config) {
         super();
+        this.projectRoot = config.projectRoot;
         this.environment = config.environment;
         this._store = new Map();
         this._normalizedIdToEntryIds = new Map();
+        this._packageMap = new Map();
+        this._runtimeMap = new Map();
```

Comment:
It will be useful for outlets. `cache.getEntriesByNormId('./components/about')` will return entries for both client and server. On the pipeline this don't meter, we just compile all of files anyway.

An outlet to write bundles will need this to write separate bundle for server and for client. The manifest on version v2 will contain both and on the server we get `main` and to serve bundles we get `browser`. Maybe `isomorphic` and `package` will never be really used, I left there mostly for debugging reasons. But maybe some crazy plugin actually wants to read more from the `package.json` than `browser` and `main` stuff.